### ISSUE TITLE
declaration-empty-line-before doesn't contain ignore option for 'first-nested'

### ISSUE THREAD

**Issue by BenjaminWFox** [stylelint/stylelint#3106]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Can't use first-nested with ignore in declaration-empty-line-before

> Which rule, if any, is this issue related to?

declaration-empty-line-before

> What CSS is needed to reproduce this issue?
```css
none - error comes from json
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "declaration-empty-line-before": ["never", {
      "ignore": ["first-nested"]
    }],
  }
}
```

> Which version of stylelint are you using?

e.g. `8.4.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

e.g. "CLI with `stylelint --config myconfig *.css`"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no

> What did you expect to happen?

rule to be applied as per documentation

> What actually happened (e.g. what warnings or errors you are getting)?

Value was flagged as an invalid option

```shell
Invalid value "first-nested" for option "ignore"...
```

**Comment by BenjaminWFox** [comment]
This is an easy fix. I have it working locally and will put in a PR.

**Comment by hudochenkov** [comment]
This option was merged 12 hours ago https://github.com/stylelint/stylelint/pull/3103 and it's not released yet.
